### ISSUE TITLE
0-conf toggle & notifications

### ISSUE THREAD

**Issue by kilrau** [BoltzExchange/boltz-web-app#346]
We moved web app to full [0-conf back in June](https://github.com/BoltzExchange/boltz-web-app/issues/92 ) to make sure reverse swaps, especially on mobile, don't fail. Users rightfully recently criticized us for that, we should offer users the option for web app to wait for one confirmation for the lockup tx with a toggle just like we had before. We also want to ask user for notification rights when toggled on and send a notification once lockup tx confirmed to open browser tab and make sure claim succeeds.

Special: on iOS browser push notifications only work for PWAs on home screen:
https://developer.apple.com/documentation/usernotifications/sending_web_push_notifications_in_web_apps_and_browsers

**Comment by kilrau** [comment]
[Here](https://www.youtube.com/watch?v=N9zpRvFRmj8) a nice video of Google explaining how sending and receiving push messages work. Important takeaway: the subscription id part of the unique endpoint url enables linking swaps that used it. Not great for user's privacy.

And obviously it won't work [on phones without Google Play Services installed](https://discuss.grapheneos.org/d/3141-how-do-push-notifications-from-websites-work/4) and we expect to have issues with privacy browsers too as they probably stubbed the push service.

Tl;dr: not looking like a viable solution to our problem of waking up user's browsers to make claiming succeed

**Comment by kilrau** [comment]
We should settle how to handle this @dni @michael1011
